app/controllers/event1/HowAddUnauthPaymentController.scala

```diff
@@ -43,21 +41,18 @@ class HowAddUnauthPaymentController @Inject()(val controllerComponents: Messages
   private val eventType = EventType.Event1
 
   def onPageLoad(waypoints: Waypoints, index: Index): Action[AnyContent] = (identify andThen getData(eventType)) { implicit request =>
-    val preparedForm = request.userAnswers.flatMap(_.get(HowAddUnauthPaymentPage(index))).fold(form)(form.fill)
-    Ok(view(preparedForm, waypoints, index))
+    Ok(view(form, waypoints, index))
```

Comment:
why did we change this with this ticket? 
https://jira.tools.tax.service.gov.uk/browse/PODS-7664

Reply:
My understanding of this is that we didn't want to pre-fill this form and thus removed the retrieval of the selected previous answer but @grantrobson to confirm

Reply:
so we removed this from all the event types' entrance page ? If so, ok. I couldn't see the requirement in ticket. If we talked before, I don't remember :(

Reply:
We don't need to save this in Mongo. It is not transformed at back end, nor is it used on any other pages. So it's just clutter.